unlock-app/src/components/creator/lock/LockIconBar.tsx

```diff
@@ -59,14 +75,3 @@ LockIconBar.propTypes = {
   withdraw: PropTypes.func.isRequired,
```

Comment:
Remove the prop `LockIconBar.propTypes` here. It's not needed if you are using typescript.